wpilibj/src/athena/java/edu/wpi/first/wpilibj/XboxController.java

```diff
@@ -0,0 +1,254 @@
+    if (hand.equals(Hand.kLeft)) {
+      return getRawAxis(1);
+    } else {
+      return getRawAxis(5);
+    }
+  }
+
+  /**
+   * This is not support for the XboxController. This method is only here to complete the GenericHID
+   * interface.
+   *
+   * @return The value of the axis (always zero)
+   */
+  @Override
+  public double getZ(Hand hand) {
+    return 0.0;
+  }
+
+  /**
+   * This is not support for the XboxController. This method is only here to complete the GenericHID
+   * interface.
+   *
+   * @return The value of the axis (always zero)
+   */
+  @Override
+  public double getTwist() {
+    return 0.0;
+  }
+
+  /**
+   * This is not support for the XboxController. This method is only here to complete the GenericHID
+   * interface.
+   *
+   * @return The value of the axis (always zero)
+   */
+  @Override
+  public double getThrottle() {
+    return 0.0;
+  }
+
+  /**
+   * Get the value of the axis.
+   *
+   * @param axis The axis to read, starting at 0.
+   * @return The value of the axis.
+   */
+  public double getRawAxis(final int axis) {
+    return m_ds.getStickAxis(m_port, axis);
+  }
+
+  /**
+   * Read the value of the bumper button on the controller.
+   *
+   * @param hand Side of controller whose value should be returned.
+   * @return The state of the button.
+   */
+  @Override
+  public boolean getBumper(Hand hand) {
+    if (hand.equals(Hand.kLeft)) {
+      return getRawButton(5);
+    } else {
+      return getRawButton(6);
+    }
+  }
+
+  /**
+   * This is not supported for the XboxController. This method is only here to complete the
+   * GenericHID interface.
+   *
+   * @param hand This parameter is ignored for the Joystick class and is only here to complete the
+   *             GenericHID interface.
+   * @return The state of the trigger (always false)
+   */
+  public boolean getTrigger(Hand hand) {
+    return false;
+  }
+
+  /**
+   * This is not supported for the XboxController. This method is only here to complete the
+   * GenericHID interface.
+   *
+   * @param hand This parameter is ignored for the Joystick class and is only here to complete the
+   *             GenericHID interface.
+   * @return The state of the top button (always false)
+   */
+  public boolean getTop(Hand hand) {
+    return false;
+  }
+
+  /**
+   * Get the button value (starting at button 1).
+   *
+   * <p>The appropriate button is returned as a boolean value.
+   *
+   * @param button The button number to be read (starting at 1).
+   * @return The state of the button.
+   */
+  public boolean getRawButton(final int button) {
+    return m_ds.getStickButton(m_port, (byte) button);
+  }
+
+  /**
+   * This is not supported for the XboxController. This method is only here to complete the
+   * GenericHID interface.
+   *
+   * @param pov The index of the POV to read (starting at 0)
+   * @return the angle of the POV in degrees, or -1 if the POV is not pressed (always -1).
+   */
+  public int getPOV(int pov) {
```

Comment:
Xbox controllers do have a POV.  It is the hat switch below the back button and the left joystick.